my-dsl.md

```diff
@@ -2,37 +2,41 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
++ I chose the SumoLogic query language
++ https://service.us2.sumologic.com/help/Default.htm#Search_Example_Cheat_Sheets.htm
+
 
 # Domain
 _Describe the language's domain in five words._
 
-
-# Computational model
-_We don't yet have a great definition of the term "computational model". 
-For now, try to come up with the clearest, most concise explanation of 
-what happens when a program in your DSL runs._
-
++ Searching / Querying application log data
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
 and why?_ 
 
++ I would classify this language as purely domain-specific because it is not turing complete. It also lacks many features that Fowler points out are necessary for a general-purpose language such as, "conditions, looping constructs and variables."
```

Comment:
Are there any features it has that might make it close to the general-purpose side?